images/router/haproxy/conf/haproxy-config.template

```diff
@@ -164,8 +189,10 @@ backend be_edge_http_{{$cfgIdx}}
 backend be_tcp_{{$cfgIdx}}
   balance leastconn
   timeout check 5000ms
+  stick-table type ip size 1m expire 5m{{ if (len $.PeerEndpoints) gt 0 }} peers openshift_peers {{ end }}
```

Comment:
Ignore a part of my above comment re: the imbalance on the haproxy backends. So we have this set for 5 mins, which seems like a short time for session affinity if sticky sessions are desired.  Maybe am not clear here on the exact use case of this - a short note would help. Thx